cmp/grid/GridModel.js

```diff
@@ -253,7 +258,8 @@ export class GridModel {
         showHover = false,
         rowBorders = false,
         cellBorders = false,
-        stripeRows = true,
+        treeGridStyleMode = 'none',
+        stripeRows = treeGridStyleMode == 'none' ? true : false,
```

Comment:
triple equals.  

Also, If we are going to support 'none' (which I support) why also support null?  Let's go ahead and use a psuedo-enumeration like RenderMode.

Reply:
I think having the enumeration in place will give us room to grow, and also obviate the need for 'mode' in the name.  Mode is confusing in this context because we already have the main flag with the name 'treeMode'

Reply:
Yes I didn't mean to insist on that word

Reply:
Got it. Didn't mean to imply insistence, just wanted to put all the info on the table